### PR TITLE
add docs about current websites

### PR DESCRIPTION
**Pull request by butlerx**

### CODE REVIEW

**Comment by VoyTechnology** [comment]
I was thinking of maybe `web` instead of www. 

Spelling CI is definatelly needed for @butlerx, but I think in the meantime he should look into https://www.grammarly.com/. It's in ads so it must be good... right?

**Comment by SeanHealy33** [comment]
Lets not devolve to ad hominem's I think it would be a cool feature is all for some digestion

**Comment by VoyTechnology** [comment]
https://github.com/inaka/spellingci

File: docs/www/techweek.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+The markdown files in the `pages` dir contains a JSON blob containing all events details.
+This blob says what template to use.
+Each page is a markdown file with a json blob at the beginning with all the details about that years talks.
+The Json is used to fill a mustache template since all the pages are the same format and layout and output to dist.
+
+The `main.js` file is responsible for displaying the countdown and loading the video feed. Also, it changes colour of the header and individual days, depending on the amount of times you have visited the page. It also handles the #Day so a specific day can be linked to.
+Gulp concats this with materilize and minifies it to `dist/js`
+
+## Updating
+There is a service set pull the lastest build from git and build it. `/etc/systemd/system/techweel.service` is as follows
```
Comment (VoyTechnology) [inline docs/www/techweek.md]: s/techweel/techweek

File: docs/www/techweek.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+```
+[Unit]
+Description=Update Techweek site off git build it
+
+[Service]
+Type=oneshot
+WorkingDirectory=/webtree/t/techwk
+ExecStart=/bin/sh -c '/usr/bin/git pull && /usr/bin/node /webtree/t/techwk/node_modules/gulp/bin/gulp.js'
+```
+and a Timer set to do this every hour at `/etc/systemd/system/techweel.timer`
```
Comment (VoyTechnology) [inline docs/www/techweek.md]: s/techweel/techweek

File: mkdocs.yml [hunk 3]
```diff
@@ -3,6 +3,9 @@ theme: readthedocs
 repo_url: https://github.com/redbrick/docs
 pages:
  - Home: index.md
+ - Web:
+   - website: www/website.md
```
Comment (VoyTechnology) [inline mkdocs.yml]: Since it is in Web category maybe move it `web` directory?

File: mkdocs.yml [hunk 4]
```diff
@@ -3,6 +3,9 @@ theme: readthedocs
...
+   - techweek: www/techweek.md
```
Comment (VoyTechnology) [inline mkdocs.yml]: Since it is in Web category maybe move it `web` directory?

**Pull request merged**